mutters-ink-bot/src/main/java/com/rabidgremlin/mutters/bot/ink/SessionUtils.java

```diff
@@ -65,6 +69,8 @@ public static void saveInkStoryState(Session session, StoryState storyState)
     try
     {
       session.setAttribute(SLOT_PREFIX + "0987654321STORYSTATE1234567890", storyState.toJson());
+      // save length of story JSON and use as a crude version check
+      session.setAttribute(SLOT_PREFIX + "0987654321STORYJSONLENGTH1234567890", Long.valueOf(storyJson.length()));
```

Comment:
Make slot suffix a final var?

Reply:
Yeah, all of that needs to be cleaned up. Will do in next big release but want to minimize change for this fix.